gradle.properties

```diff
@@ -14,3 +14,6 @@ POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=square
 POM_DEVELOPER_NAME=Square, Inc.
+
+org.gradle.parallel=true
```

Comment:
This should not be committed because it breaks the release process. Sonatype sees concurrent uploads as being separate.